### ISSUE TITLE
Remove redirects from the Pulse Accessibility scans 

### ISSUE THREAD

**Issue by smarina04** [18F/pulse#502]
Per OGP's decision as of 8/12/16, all redirects should be removed from the list of domains.

**Comment by gbinal** [comment]
Roger.  This'll be included in #477.

**Comment by gbinal** [comment]
done
